package.json

```diff
@@ -13,6 +13,7 @@
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.17.9",
+    "@chakra-ui/cli": "0.0.0-pr-20211126153854",
```

Comment:
Can we go back to a conversation about this? 😅 

Is it really necessary in `components` repo? When running storybook i can`t see the error that you mentioned which is `TypeError: Cannot read properties of undefined (reading 'message')` - I think it's more related to `token-dashboard` and I'm not sure if `components` repo also has something to do with that error 🤔